### PR TITLE
Treeview tests

### PR DESCRIPTION
**Pull request by sseliverstov**

### CODE REVIEW

File: allure-generator/src/test/javascript/bootstrap.js [hunk 1]
```diff
@@ -1,3 +1,11 @@
+import {Behaviors} from 'backbone.marionette';
+
+window.Marionette = require('backbone.marionette');
```
Comment (just-boris) [inline allure-generator/src/test/javascript/bootstrap.js]: Is that required? Seems like it should not.

Comment (sseliverstov) [inline allure-generator/src/test/javascript/bootstrap.js]: it is your magic =) without it I had exception: " TypeError: BehaviorClass is not a constructor
"

Comment (just-boris) [inline allure-generator/src/test/javascript/bootstrap.js]: Ok, we need to update one more line. Here is the entire code:

```
import {Behaviors, Behavior} from 'backbone.marionette';

Behaviors.behaviorsLookup  = function(options, key) {
 return { [key]: Behavior };
}
```

**Comment by sseliverstov** [comment]
Tnx @just-boris, I fix it. @baev, merge?

**Comment by just-boris** [comment]
Jenkins is failing

File: allure-generator/src/test/javascript/components/tree.spec.js [hunk 2]
```diff
@@ -74,17 +73,12 @@ describe('Tree', function () {
...
-        this.el = renderView(data);
-        expect(this.el.paneTitle()).toMatch(tabName);
-    });
 
     describe('with empty data', function() {
 
         it('should render with empty data', function () {
             const el = renderView([]);
-            expect(el.tree().text().trim()).toMatch('There are no items');
+            expect(el.tree().text().trim()).toMatch('');
```
Comment (just-boris) [inline allure-generator/src/test/javascript/components/tree.spec.js]: it's better to assert that there are no nodes inside. Like so:

```js
expect(page.nodes()).toHaveLength(0)
```

**Comment by sseliverstov** [comment]
Yep, sorry for the spam =(

**Comment by sseliverstov** [comment]
@just-boris I fix it =)

**Comment by just-boris** [comment]
happy to see that.
let's go forward!

**Comment by baev** [comment]
thanks @sseliverstov @just-boris

**Pull request merged**